hw_recogs.ipynb

```diff
@@ -1354,7 +1354,7 @@
    "id": "e7a19b17-f58a-4a40-a07a-71f362dd33d1",
    "metadata": {},
    "source": [
-    "## Question 3: In-context learning with DSP [2 points]"
+    "## Question 3: In-context learning with DSPy [2 points]"
    ]
   },
   {
```

Comment:
From "where you have a working DSP program" to "where you have a working DSPy program"

Reply:
Thanks!